src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -129,19 +129,20 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
         <small className="title-reports-box">
           <FormattedMessage id="reports_daily_visits.title" />
         </small>
-        <S.ContentContainer>
-          {state.loading ? (
-            <Loading />
-          ) : !state.chartData ? (
+
+        {state.loading ? (
+          <Loading />
+        ) : !state.chartData ? (
+          <S.ContentContainer>
             <BoxMessage className="dp-msj-error bounceIn">
               <p>
                 <FormattedMessage id="trafficSources.error" />
               </p>
             </BoxMessage>
-          ) : (
-            <C3Chart config={chartConfig} dataOptions={chartDataOptions} data={state.chartData} />
-          )}
-        </S.ContentContainer>
+          </S.ContentContainer>
+        ) : (
+          <C3Chart config={chartConfig} dataOptions={chartDataOptions} data={state.chartData} />
```

Comment:
Just to understand, what is the rationale for removing the container in this chart?

Reply:
the container it's only for the box message. The chart has your container